projects/vdk-core/src/vdk/internal/builtin_plugins/builtin_hook_impl.py

```diff
@@ -58,31 +62,50 @@ def vdk_initialize(self, context: CoreContext) -> None:
 
         Configuration can override an attempt/op/execution id.
 
-        Otherwise each is auto-generated in format:
-        * Attempt id format is "execution_id-random_suffix"
-        * Execution id format is "op_id-random_suffix"
+        Attempt id and Execution id are extracted from env variables.
+        IF not present they are auto-generated.
+
+        If auto generated:
```

Comment:
nit: 'auto-generated' in one place but 'auto generated' in another.
```suggestion
        If auto-generated:
```